src/cecs277TermProject/Logger.java

```diff
@@ -0,0 +1,28 @@
+package cecs277TermProject;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+
+/**
+ * Logger is simply our file I/O class, meant to write to the transaction log.
+ */
+public class Logger 
```

Comment:
This class should be a singleton. You should implement singleton pattern for this class.